engine.go

```diff
@@ -238,8 +238,12 @@ func (e *EngineBackend) ForkchoiceUpdatedV1(ctx context.Context, heads *types.Fo
 	binary.BigEndian.PutUint64(id[:], idU64)
 
 	plog := e.log.WithField("payload_id", id)
-	plog.WithField("attributes", attributes).Info("Preparing new payload")
-
+	plog.WithFields(logrus.Fields{
+		"timestamp":              attributes.Timestamp,
```

Comment:
```suggestion
		"timestamp":               attributes.Timestamp,
```
missing a space now :)

Reply:
thanks!